time-series-forecast/src/steps/transformer.py

```diff
@@ -0,0 +1,42 @@
+from zenml.steps import step, Output
+import pandas as pd
+import numpy as np
+from sklearn.model_selection import train_test_split
+
+@step
+def transformer(data: pd.DataFrame) -> Output(
+    X_train=np.ndarray, X_test=np.ndarray, y_train=np.ndarray, y_test=np.ndarray
+):
+    df = data.copy()
+    cardinal_directions = {'N': 0.0,
```

Comment:
I changed the code a bit so it looks better and added a description of how the feature engineering works. I find it more clear if the code stays together instead of using a config file in this case